PackageViewModel/PublishPackage/PublishPackageViewModel.cs

```diff
@@ -23,6 +23,8 @@ public sealed class PublishPackageViewModel : ViewModelBase, IObserver<int>, IDi
         private string _publishKey;
         private bool? _publishAsUnlisted = true;
         private bool? _appendV2ApiToUrl = true;
+        private bool? _useApiKey = true;
+        private bool? _unlistPreviousVersions = false;
```

Comment:
why do we need this change?

Reply:
We don't. I started working on another issue when I found the vsts bug. That `_unlistPreviousVersions` shouldn't be there.

I'll remove it

Reply:
👍